### DISCUSSION TITLE
how delete extra space at the end of an unpaired tag while formatting?

### DISCUSSION THREAD

**Discussion by IngWonk** [prettier/prettier-vscode#2972] [Q&A]
when I format html code with prettier, it adds an unnecessary space when closing an unpaired tag like a link or meta tags. How can this be avoided? I couldn't find such an option in the extension settings

![p](https://user-images.githubusercontent.com/112816570/233424946-0d178f13-4c5e-4111-94b5-669efb858820.png)

**Comment by gosha5803** [comment]
Did you find the solution?
